src/components/Me/Navigation/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Navbar.css';
+
+const Navbar = () => {
+  return (
+    <div>
+      <div className="menu-tablet">
+        <div className="logo-left">
+          <img
+            class="sidenav-logo"
+            src={`${process.env.PUBLIC_URL}/codingcoach-logo-192.png`}
+            alt=""
+          />
+        </div>
+        <div className="house side-icon">
+          <img src={`${process.env.PUBLIC_URL}/images/icons/Home.png`} alt="" />
```

Comment:
Please use the svg files I sent you through slack.

Reply:
I don't have svg for mentors icon.
It's not in the slack also.